docs/deployment/pull-backup.rst

```diff
@@ -0,0 +1,92 @@
+.. include:: ../global.rst.inc
+.. highlight:: none
+
+Backing up in pull mode
+=======================
+
+Assume you have a pull backup system set up with borg, where a backup server
+pulls the data from the target via sshfs.
+
+In this mode, the backup client's file system is mounted via sshfs on the backup
+server. Pull mode is even possible if the SSH connection must be established by
+the client via remote tunnel.
+
+Creating backup
+---------------
+
+In this approach the client file system is simply mounted and then backed up.
+
+::
+
+    mkdir /mnt/sshfs
+    sshfs root@TARGET_IP:/ /mnt/sshfs
+    borg create /REPOSITORY::archive /mnt/sshfs
```

Comment:
iirc we have some convention for a repo path and iirc it is not `/REPOSITORY`, please use the same you see elsewhere in the docs.

this also applies for other stuff like hosts, etc.

Reply:
I'll have a look...